content/influxdb/cloud/reference/cli/influx/v1/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: influx v1
+description: >
+  The `influx v1` command provides commands for working with the InfluxDB 1.x API in InfluxDB 2.0.
+menu:
+  influxdb_cloud_ref:
+    name: influx v1
+    parent: influx
+weight: 101
+influxdb/v2.0/tags: []
```

Comment:
- Update `influxdb/v2.0/tags` to `influxdb/cloud/tags` (Make this change on all the cloud pages.
- If there aren't any tags, remove this from the frontmatter. The empty array will tell Hugo that tags exist and the template will render them, but it will just look like awkward, out of place space.